### PR TITLE
update TOP and add toptable

### PR DESCRIPTION
**Pull request by saradbowman**

### CODE REVIEW

File: www/toptable.mako [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+        <!--End Container-->
+    </div>
+    <!-- END PAGE CONTAINER -->
+
+    <!-- BEGIN PAGE LEVEL JAVASCRIPTS(REQUIRED ONLY FOR CURRENT PAGE) -->
+    <%def name="javascript_bottom()">
+
+            <script type="text/javascript" src="../static/plugins/fancybox/source/jquery.fancybox.pack.js"></script>
+            <script src="../static/plugins/jquery.mixitup.min.js"></script>
+            <script src="../static/scripts/app.js"></script>
```
Comment (erinspace) [inline www/toptable.mako]: you don't need this line - and it will throw javascript errors in the console!

File: www/toptable.mako [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+            <script src="../static/scripts/portfolio.js"></script>
+                        <script type="text/javascript">
+                jQuery(document).ready(function() {
```
Comment (erinspace) [inline www/toptable.mako]: same with this JQuery stuff - this was copied from another mako file and does nothing here

File: www/toptable.mako [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+    </div>
+    <!-- END PAGE CONTAINER -->
+
+    <!-- BEGIN PAGE LEVEL JAVASCRIPTS(REQUIRED ONLY FOR CURRENT PAGE) -->
+    <%def name="javascript_bottom()">
+
+            <script type="text/javascript" src="../static/plugins/fancybox/source/jquery.fancybox.pack.js"></script>
+            <script src="../static/plugins/jquery.mixitup.min.js"></script>
+            <script src="../static/scripts/app.js"></script>
+            <script src="../static/scripts/portfolio.js"></script>
```
Comment (erinspace) [inline www/toptable.mako]: using this anywhere?

**Comment by saradbowman** [comment]
I think I fixed it!

File: www/toptable.mako (hunk 1 above)
Comment (erinspace) [inline www/toptable.mako]: So close - this line too!

**Comment by saradbowman** [comment]
maybe?

**Comment by erinspace** [comment]
beautiful!

**Pull request merged**